### PR TITLE
Add query for checking time since last activity on PRs

### PR DESCRIPTION
**Pull request by colebow**
Output:
image

### CODE REVIEW

**Comment by colebow** [comment]
@nineinchnick I'm not able to add you as a reviewer but if you could take a look at this when you get a chance

File: sql/pr/inactivity-on-prs.sql [hunk 1]
```diff
@@ -0,0 +1,110 @@
+-- Time PRs have gone without any updates
```
Comment (nineinchnick) [inline sql/pr/inactivity-on-prs.sql:1]: This title is wrong

File: sql/pr/inactivity-on-prs.sql [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    '3 to 7 days',
+    '7 to 14 days',
+    '14 to 21 days',
+    '21 to 30 days',
+    '30 to 90 days',
+    'over 90 days'],
+    bucket
+    ) AS range,
+    sum (value) AS value,
+    FLATTEN(ARRAY_AGG(num)) AS num
```
Comment (nineinchnick) [inline sql/pr/inactivity-on-prs.sql:97]:
```suggestion
    ARRAY_SORT(FLATTEN(ARRAY_AGG(num))) AS num
```

File: sql/pr/inactivity-on-prs.sql [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    bucket
+ORDER BY
+    bucket
+    )
+SELECT
+    *
+FROM
+    grouped
+ORDER BY
+    bucket;
```
Comment (nineinchnick) [inline sql/pr/inactivity-on-prs.sql:110]: Add an EOL at EOF

File: sql/pr/inactivity-on-prs.sql [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    FLATTEN(ARRAY_AGG(num)) AS num
+FROM
+    histogram
+GROUP BY
+    bucket
+ORDER BY
+    bucket
+    )
+SELECT
+    *
```
Comment (nineinchnick) [inline sql/pr/inactivity-on-prs.sql:106]: Don't select the `bucket` column here, it's only used for ORDER BY and doesn't make sense to the user.

File: sql/pr/inactivity-on-prs.sql [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+          AND repo = 'trino'
+          AND state = 'open'
+    ),
+    last_review_comments AS (
+        SELECT
+            r.pull_number,
+            CAST (CURRENT_TIMESTAMP AT TIME ZONE 'UTC' AS TIMESTAMP) - max(r.submitted_at) AS time_since_review
+        FROM
+            pulls p
+                JOIN reviews r ON (r.owner, r.repo, r.pull_number) = (p.owner, p.repo, p.pull_number)
```
Comment (nineinchnick) [inline sql/pr/inactivity-on-prs.sql:23]: nit: this could be a single line, same in other CTEs

**Review by nineinchnick — CHANGES_REQUESTED** [review]
If you want it to appear on the reports page, add it in the right place here: https://github.com/nineinchnick/trino-cicd/blob/main/.github/workflows/reports.yml#L51

File: sql/pr/inactivity-on-prs.sql (hunk 4 above)
Comment (nineinchnick) [reply sql/pr/inactivity-on-prs.sql:106]: Also use more human friendly aliases (double quote them if you need to use spaces)

File: sql/pr/inactivity-on-prs.sql (hunk 5 above)
Comment (colebow) [reply sql/pr/inactivity-on-prs.sql:23]: this looks consistent w/the rest of the query and the prettify from the SQL formatter?

**Review by nineinchnick — APPROVED** [review]

**Pull request merged**